### ISSUE TITLE
[QUESTION] Detect `showModal` method from `dialog` element

### ISSUE THREAD

**Issue by KirianCaumes** [amilajack/eslint-plugin-compat#616]
Hi,
Currently, if I write the following code, the `showModal` method is not detected as a problem:
```js
const dialog = document.createElement('dialog')
dialog.showModal()
```
With this `.browserslistrc` configuration:
```
last 10 ie versions
```
But according to caniuse.com, it is indeed not compatible: https://caniuse.com/mdn-api_htmldialogelement_showmodal

Is this a bug or something that is not currently in place in the library? Is it even possible to detect?
In any case, thank you for this awesome librairie 😁

**Comment by replete** [comment]
https://developer.mozilla.org/en-US/docs/Web/API/HTMLDialogElement/showModal
Compat at the bottom
